src/unit/get.js

```diff
@@ -0,0 +1,76 @@
+define([
+	"./categories"
+], function( unitCategories ) {
+
+function stripPluralGarbage( data ) {
+	var aux, pluralCount;
+
+	if ( data ) {
+		aux = {};
+		for ( pluralCount in data ) {
+			aux[ pluralCount.replace( /unitPattern-count-/, "" ) ] = data[ pluralCount ];
+		}
+	}
+
+	return aux;
+}
+
+/**
+ * get( unit, form, cldr )
+ *
+ * @unit [String] The full type-unit name (eg. duration-second), or the short unit name
+ * (eg. second).
+ *
+ * FIXME
+ *
+ * Return the plural map of a unit, eg: "second"
+ * { "one": "{0} second",
+ *   "other": "{0} seconds" }
+ * }
+ *
+ * Or the Array of plural maps of a compound-unit, eg: "foot-per-second"
+ * [ { "one": "{0} foot",
+ *     "other": "{0} feet" },
+ *   { "one": "{0} second",
+ *     "other": "{0} seconds" } ]
+ *
+ * Or undefined in case the unit (or a unit of the compound-unit) doesn't exist.
+ */
+var get = function( unit, form, cldr ) {
+	var ret;
+
+	// Get unit or <type>-unit (eg. "duration-second").
+	[ "" ].concat( unitCategories ).some(function( category ) {
+		return ret = cldr.main([
+			"units",
+			form,
+			category.length ? category + "-" + unit : unit
+		]);
+	});
+
+	// Rename keys s/unitPattern-count-//g.
+	ret = stripPluralGarbage( ret );
+
+	// Compound Unit, eg. "foot-per-second" or "foot/second".
+	if ( !ret && ( /-per-|\// ).test( unit ) ) {
```

Comment:
I like allowing the use of `/` as well as `-per-`. But, `-per-` is what is going to be found on CLDR. So, in the beginning of this function we need `unit = unit.replace( /\// , "-per-" );`, otherwise `kilometer/hour` would not use the precomputed `kilometer-per-hour` form and it would get both recursively only.